src/calculateRentalCost.js

```diff
@@ -4,7 +4,16 @@
  * @return {number}
  */
 function calculateRentalCost(days) {
-  // write code here
+  const dailyRate = 40;
+  let totalCost = days * dailyRate;
+
+  if (days >= 7) {
+    totalCost -= 50;
+  } else if (days >= 3) {
+    totalCost -= 20;
+  }
+
+  return totalCost;
```

Comment:
The function name 'calculateRentalCost' is descriptive and clearly states its purpose, which is good. However, the implementation uses hardcoded values for the daily rate, and the discounts for different rental durations. These should be replaced with appropriately named constants to improve code readability and maintainability.